ui/server/src/main/scala/pl/touk/nussknacker/ui/api/AppResources.scala

```diff
@@ -26,22 +26,27 @@ class AppResources(config: Config,
                    processRepository: FetchingProcessRepository[Future],
                    processValidation: ProcessValidation,
                    jobStatusService: JobStatusService)(implicit ec: ExecutionContext)
-  extends Directives with FailFastCirceSupport with LazyLogging with RouteWithUser with SecurityDirectives {
+  extends Directives with FailFastCirceSupport with LazyLogging with RouteWithUser with RouteWithoutUser with SecurityDirectives {
+
+  //Public app resources
+  def route(): Route = pathPrefix("app") {
```

Comment:
maybe let's name this routes: publicRoute, securedRoute instead of giving comment. I know that there is implicit in second version but in invocation it looks the same... Also still both routes have the same prefix. Hot it supposed to work?

Reply:
Akka http has join paths - at end everything is works fine.

Reply:
Changed. Now we have publicPublic and securedRoute.

Reply:
Now comment is not needed, but we can leave it just in case ;-)

Reply:
Removed ;)